drivers/net/mvpp2/mrvl_ethdev.c

```diff
@@ -2875,6 +2907,7 @@ mrvl_tx_pkt_burst(void *txq, struct rte_mbuf **tx_pkts, uint16_t nb_pkts)
 		 * in case unsupported ol_flags were passed
 		 * do not update descriptor offload information
 		 */
+		mvpp2_tx_hw_tstamp(q->priv, &descs[i], mbuf);
```

Comment:
I think that this function should be inlined as it is in the fast path (in the future, we probably need to consider inlining MUSDK calls too).
Also, we should think how to minimize performance impact on non-ptp usages.

Reply:
done

Reply:
For RX TS per-port config ol_flag is used to enable, for TX TS per-packet ol_flag is used to perform stamping